recipe/meta.yaml

```diff
@@ -65,12 +65,17 @@ requirements:
     - fontconfig
     - libcurl
     - gettext  # [osx]
+    - nss >=3.19
+    - boost >=1.58.0
+    - lcms2
+    - poppler-data  # [unix]
+    - qt >=5.9
   run:
     - poppler-data
-    - cairo  # [not win]
+    - cairo >=1.10.0  # [not win]
```

Comment:
These explicit run constrains, especially with version bounds, should not be necessary; they should be automatically generated by the `run_exports` of the `build` dependencies.